updated sbt 1.0.x

magnolia-k opened the pull request.

xuwei-k commented:
```
Downloading sbt launcher for 1.0.2:
 From https://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/1.0.2/sbt-launch.jar
 To /home/travis/.sbt/launchers/1.0.2/sbt-launch.jar
Download failed. Obtain the jar manually and place it at /home/travis/.sbt/launchers/1.0.2/sbt-launch.jar
```


https://travis-ci.org/scalatra/scalatra/jobs/290985387

We should update the `sbt` shell script in scalatra repo.
- https://github.com/scalatra/scalatra/blob/dd1d114f4f4/sbt
- https://github.com/paulp/sbt-extras/commit/16f10bec81655453a6531114c2c40de7a3eb5ee8

or just remove `sbt` script and use travis-ci default sbt?

magnolia-k commented:
The start script of sbt in the repository did not correspond to sbt 1.0.x (`sbt-launch.jar` can not be downloaded).

I changed travis.yml so that use sbt which is installed in the environment beforehand and deleted the start script of sbt in the repository.

It is more appropriate to let the OS package manager to install sbt.

xuwei-k commented on build.sbt:
```diff
@@ -39,16 +39,15 @@ lazy val scalatraSettings = Seq(
-    name := "scalatra-unidoc",
+  base = file(".")).settings(
+    scalatraSettings ++ Seq(
+    name := "scalatra-project",
```
Is there any reasons change the name?

magnolia-k replied:
In sbt 1.0.x and later, the project name is output before the command prompt, but because `scalatra-unidoc` is displayed as the default project, it is hard to understand.

xuwei-k replied:
This is artifactId for unidoc. try `sbt publishLocal`

```

[info] Packaging /Users/kenji/scalatra/target/scala-2.12/scalatra-unidoc_2.12-2.6.0-SNAPSHOT-javadoc.jar ...
[info] Done packaging.
[info] 	published scalatra-unidoc_2.12 to /Users/kenji/.ivy2/local/org.scalatra-unidoc_2.12/2.6.0-SNAPSHOT/docs/scalatra-unidoc_2.12-javadoc.jar
[info] 	published scalatra-unidoc_2.12 to /Users/kenji/.ivy2/local/org.scalatra-unidoc_2.12/2.6.0-SNAPSHOT/poms/scalatra-unidoc_2.12.pom
[info] 	published ivy to /Users/kenji/.ivy2/local/org.scalatra-unidoc_2.12/2.6.0-SNAPSHOT/ivys/ivy.xml
```

```

```

xuwei-k replied:
You can customize sbt prompt 

- https://github.com/sbt/sbt/blob/v1.0.2/main/src/main/scala/sbt/Keys.scala#L128
- default https://github.com/sbt/sbt/blob/v1.0.2/main/src/main/scala/sbt/Defaults.scala#L2966-L2972

magnolia-k replied:
thanks!

I fixed it!

magnolia-k commented:
@takezoe @xuwei-k 

Is it okay to merge?
